### PR TITLE
Post Content Processing

### PR DESCRIPTION
**Pull request by enggabhishek**
Added the modified version of PII removal code

### CODE REVIEW

File: api/shield_net/services/__init__.py [hunk 1]
```diff
@@ -7,7 +8,8 @@ def post_process_content(content: str) -> str:
     :param content: The raw content generated by the LLM.
     :return: The processed content.
     """
-    # Add custom post-processing logic here if needed
+    replacePII = ReplacePII()
```
Comment (liranzxc) [inline api/shield_net/services/__init__.py]: @enggabhishek you init the class each call function.  It's not good,  pass the class instance as argument

**Review by liranzxc — CHANGES_REQUESTED** [review]

File: api/shield_net/services/__init__.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-def post_process_content(content: str) -> str:
+def post_process_content(content: str, replace_pii: ReplacePII) -> str:
     """
     Applies post-processing to the LLM's response content.
 
     :param content: The raw content generated by the LLM.
     :return: The processed content.
     """
-    # Add custom post-processing logic here if needed
+    content = replace_pii.execute_pii_removal(content)
```
Comment (liranzxc) [inline api/shield_net/services/__init__.py:10]: @enggabhishek  you run the code ? 

execute_pii_removal is not a static method .. you need to create a instance of the class

**Review by liranzxc — CHANGES_REQUESTED** [review]

File: api/shield_net/services/__init__.py (hunk 1 above)
Comment (enggabhishek) [reply api/shield_net/services/__init__.py]: Updated the code as per your suggestion.

File: api/shield_net/services/__init__.py (hunk 2 above)
Comment (enggabhishek) [reply api/shield_net/services/__init__.py:10]: I have imported instance instead of ReplacePII class in __init__.py file as a global variable.

**Pull request closed without merging**